src/panels/config/devices/device-detail/integration-elements/mqtt/mqtt-messages.ts

```diff
@@ -50,7 +51,7 @@ class MQTTMessages extends LitElement {
                 (message) => html`
                   <li class="message">
                     <div class="time">
-                      Received
+                      ${this.direction}
```

Comment:
You could have kept it 1 element and add a `direction` property to it 😄

Reply:
Fixed :)